src/date-range-picker/index.tsx

```diff
@@ -266,7 +256,7 @@ const DateRangePicker = React.forwardRef(
           dropdownId={dropdownId}
         >
           {isDropDownOpen && (
-            <DateRangePickerDropdown
+            <DateRangePickerEditor
```

Comment:
I thought the editor is being used by both embedded and original components. If that's not the case, I don't see a reason to make a dropdown -> editor rename.